sdk/src/main/scala/io/horizen/account/state/events/AddCrossChainRedeemMessage.scala

```diff
@@ -0,0 +1,36 @@
+package io.horizen.account.state.events
+
+import io.horizen.account.sc2sc.{AccountCrossChainMessage, AccountCrossChainMessageSerializer}
+import io.horizen.account.state.events.annotation.{Indexed, Parameter}
+import org.web3j.abi.datatypes.DynamicBytes
+import org.web3j.abi.datatypes.generated.Bytes32
+
+import scala.annotation.meta.getter
+
+case class AddCrossChainRedeemMessage(
+                                       @(Parameter @getter)(1) @(Indexed @getter) accountCrossChainMessage: DynamicBytes,
```

Comment:
I'm not sure about the benefit of having `accountCrossChainMessage bytes` indexed.
What can be useful is to keep `accountCrossChainMessage` in a context form and index `receiver` and sender sidechain id fields.
Then the user can easily filter the log by his address to see the Redeems for his account, for example. 

Other fields, like `proof`, have no sense to be put into the Log.